languages/rust/exercises/concept/string-vs-str/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+`String` is a potentially-mutable utf8-encoded representation of a sequence of Unicode codepoints. (In Java or C#, this type is called `StringBuilder`.)
```

Comment:
> (In Java or C#, this type is called `StringBuilder`.)

This bit will only really be useful to Java or C# programmers, which means that it won't benefit students not familiar with those languages. It can be very useful information, so you could:

1. Leave it as is
2. Rewrite it slightly, to something like: ```In some languages, this type is referred to as a `StringBuilder`.```

Whatever you prefer :)